packages/engine/errors/2571.md

```diff
@@ -0,0 +1,11 @@
+---
+original: "Object is of type 'unknown'."
+excerpt: "This variable is unknown so I don't know what type it or if {0} is in it."
+---
+
+This variable is unknown so I don't know what type it or if
+
+```
+{0}
```

Comment:
As you can see in the original, there isn't a `{0}` to reference here. This means that both the excerpt and body need changing to be clearer about what the intention of the error is.

Reply:
Yeah, my bad. I've already updated and removed the reference in the excerpt and body since I couldn't really get much details from the original error.